internal/rpc/rpcserver/server.go

```diff
@@ -2420,13 +2420,21 @@ func (t *ticketbuyerV2Server) RunTicketBuyer(req *pb.RunTicketBuyerRequest, svr
 	var votingAddress dcrutil.Address
 	var err error
 	if req.VotingAddress != "" {
+		if req.VspHost != "" || req.VspPubkey != "" {
+			return status.Errorf(codes.InvalidArgument,
+				"request contains both legacy stakepoold and vspd options.")
+		}
```

Comment:
the voting address can be specified without using a stakepool.  Only the check with the pool address (where stakepool fees are paid) is needed.